app/views/_create-secret.html

```diff
@@ -0,0 +1,267 @@
+    <div class="form-group">
+      <label for="dockerServer" class="required">Docker Registry Server Address</label>
+      <div>
+        <input class="form-control"
+          id="username"
+          name="username"
+          ng-model="newSecret.data.dockerServer"
+          type="text"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
+          required>
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label for="dockerUser" class="required">Docker Username</label>
+      <div>
+        <input class="form-control"
+          id="username"
+          name="username"
+          ng-model="newSecret.data.dockerUsername"
+          type="text"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
+          required>
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label for="dockerPassword" class="required">Docker Password</label>
+      <div>
+        <input class="form-control"
+          id="username"
+          name="username"
+          ng-model="newSecret.data.dockerPassword"
+          type="password"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
+          required>
+      </div>
+    </div>
+
+    <div class="form-group">
+      <label for="dockerEmail" class="required">Docker Email</label>
+      <div>
+        <input class="form-control"
+          type="email"
+          id="passwordToken"
+          name="passwordToken"
+          ng-model="newSecret.data.dockerMail"
+          type="password"
+          autocorrect="off"
+          autocapitalize="off"
+          spellcheck="false"
+          required>
+      </div>
+    </div>
+  </div>
+
+  <div ng-if="newSecret.authType === 'Docker Config'">
+    <div class="form-group" id="docker-config">
+      <label for="dockerConfig" class="required">Docker Configuration File</label>
+      <osc-file-input 
+        model="newSecret.data.dockerConfig"
+        drop-zone-id="docker-config"
+        dragging="false"
+        help-text="Upload .dockercfg or config.json file"
+        ng-disabled="false"
+        show-values="false"></osc-file-input>
+      <div ui-ace="{
+        mode: 'json',
+        theme: 'eclipse',
+        onLoad: aceLoaded,
+        rendererOptions: {
+          fadeFoldWidgets: true,
+          showPrintMargin: false 
+        }
+      }" ng-model="newSecret.data.dockerConfig" class="create-secret-editor ace-bordered yaml-mode" id="dockerconfig-editor" required></div>
+    </div>
+  </div>
+
+  <div class="form-group">
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" ng-model="newSecret.linkSecret"/>
+        Link with secret with Service Account
+        <span class="help action-inline">
+          <a href>
+            <i class="pficon pficon-help" aria-hidden="true"
+            data-toggle="tooltip" data-original-title="Linking tooltip">
```

Comment:
"Linking tooltip" ? :)

I think this is better as help text under the checkbox.